packages/web/src/components/collection/CollectionCard.test.tsx

```diff
@@ -149,4 +149,14 @@ describe('CollectionCard', () => {
       })
     )
   })
+
+  it('premium collections owned by user are rendered correctly', () => {
+    renderCollectionCard({ reduxState: { account: { userId: 2 } } })
+
+    expect(
+      screen.getByRole('button', {
+        name: 'Test Collection Test User Reposts 10 Favorites 5'
+      })
+    ).toBeInTheDocument()
```

Comment:
is there a way to expect the locked icon to not exist?

Reply:
yeah if you notice i changed regex to quotes, which exactly matches, ie locked icon not in there. since the locked icon's title is not present, ie it's not rendered

Reply:
a bit round-about though, so ill add a comment